src/App.tsx

```diff
@@ -1,24 +1,52 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock/Clock';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number | string;
```

Comment:
Imo, I would suggest just using number type. The string is redundant here since you're using Math.random() to generate the random name.

Reply:
I wanted to use `'Try to set my name!'` string like starting name but maybe it was not a good idea 👀